### PR TITLE
[Hotfix] Ghosts can't fuck with fishtanks

### LINKED ISSUE #17123: GHOSTS CAN TAP ON WALL AQUARIUMS
(WEB REPORT BY: publicenemymaipai REMOTE: 172.93.109.202:7777)
# Revision
c260ac9f653fd3cbd418d08d88a9fd99c8709513

# Description
AHHHHHHHHHHHHHHHHHHHHHHHHHH
# Steps to Reproduce

# What you Expected

# What Actually Happened

### PR DESCRIPTION
**Pull request by ShiftyRail**
Closes #17123

### CODE REVIEW

**Comment by ShiftyRail** [comment]
For the sake of atomicity I'll make the thing about hurt intent doing no damage in another, more fleshed-out PR.

File: code/modules/fish/fish_tank.dm [hunk 1]
```diff
@@ -541,6 +541,9 @@
 
 /obj/machinery/fishtank/attack_hand(var/mob/user)
 
+	if (isobserver(user))
```
Comment (Intigracy) [inline code/modules/fish/fish_tank.dm]: `if(!Adjacent(user) || user.incapacitated())`

Comment (ShiftyRail) [reply code/modules/fish/fish_tank.dm]: Dear God, thanks

**Comment by MadmanMartian** [comment]
so, do ghosts fail incapacitated or adjacent checks?

**Comment by ShiftyRail** [comment]
Ghosts do, I'm pretty sure. Ghost have a stat of 2, which is dead.

**Comment by ShiftyRail** [comment]
Thanks Inti

**Pull request merged**